upup/pkg/fi/cloudup/tagbuilder.go

```diff
@@ -60,6 +60,10 @@ func buildCloudupTags(cluster *api.Cluster) (sets.String, error) {
 
 	case api.CloudProviderOpenstack:
 
+	case api.CloudProviderALI:
+		{
+			tags.Insert("_ali")
```

Comment:
I hope you don't need this - we're trying to get rid of the tags, and I don't see you using them anywhere :-)

But ... it's pretty harmless!